application-templates/starter/.env

```diff
@@ -1,2 +1,3 @@
 ENABLE_NEW_JSX_TRANSFORM="true"
-FAST_REFRESH="true"
\ No newline at end of file
+ENABLE_I18N_MESSAGE_COMPILATION="true"
```

Comment:
We currently have 4 templates:
* Custom Applications - Javascript
* Custom Applications - Typescript
* Custom Views - Javascript
* Custom Views - Typescript

Perhaps it would be better to have the new compilation enabled for one template of Custom Applications and another one for Custom Views so we keep at least the other two with the default behaviour and we cover both cases in both extensions types.

What do you think?

Reply:
Wanted to go into the opt-out direction. I don't think there is a good reason to keep the messages in extracted object shape in the production bundle.
But there is no reason not to mix and match, I can move the `structured_json` case from custom app ts to custom view ts.

Reply:
@CarlosCortizasCT now we mix and match

Reply:
Maybe double negative got lost in the translation, I agree that having coverage on the custom views is a good idea. Hesitated to touch them, as a bit unfamiliar at this point 😅 

Yes this could be frustrating, will add the suggestion to use `DISABLE_I18N_MESSAGE_COMPILATION` on the error message of the compiler when doing the opt-out transition.